src/Thermodynamics/dynamic_states.jl

```diff
@@ -9,9 +9,9 @@ struct PotentialTemperatureState{FT} <: AbstractThermodynamicState{FT}
     reference_pressure :: FT
 end
 
-@inline is_absolute_zero(𝒰::PotentialTemperatureState) = 𝒰.potential_temperature == 0
+@inline is_absolute_zero(𝒰::LiquidIcePotentialTemperatureState) = 𝒰.potential_temperature == 0
```

Comment:
Where is `LiquidIcePotentialTemperatureState` supposed to be defined?

Reply:
It was supposed to be a rename of `PotentialTemperatureState`, but I forgot to rename everything. Although maybe I should have asked @kaiyuan-cheng if this was a good idea